public/locales/cs/translation.json

```diff
@@ -555,7 +556,8 @@
 		"Sessions": "Relace",
 		"Roles": "Role",
 		"Resources": "Zdroje",
-		"Clients": "Klienti"
+		"Clients": "Klienti",
+		"Home": "Domovská stránka"
```

Comment:
@Aringoaway Well, not here. Only for Breadcrumbs

Reply:
Here it will be `Domů`

Reply:
@Pe5h4 Why shouldn't it be added here? In my opinion this should translate into a Sidebar too

Reply:
@Aringoaway Because in sidebar we want to have shorter names

Reply:
So here it will be `Home: Domů`